Code/GraphMol/Kekulize.cpp

```diff
@@ -460,7 +460,12 @@ void kekulizeFused(RWMol &mol, const VECT_INT_VECT &arings,
     // number of backTracks) and we still need to backtrack
     // can't kekulize this thing
     std::ostringstream errout;
-    errout << "Can't kekulize mol " << std::endl;
+    errout << "Can't kekulize mol.";
+    errout << "  Atoms remaining:";
+    for (unsigned int i = 0; i < nats; ++i) {
+      if (dBndCands[i]) errout << " " << i;
```

Comment:
Do you mean you don't want any kind of message about the kekulization error or that you just don't want the atom indices but the "Can't kekulize mol." bit is ok?